riotfile.py

```diff
@@ -711,20 +711,19 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                     pys=select_pys(),
                     pkgs={
                         "elasticsearch": [
-                            "~=7.6.0",
-                            "~=7.8.0",
-                            "~=7.10.0",
-                            # latest,
-                            # FIXME: Elasticsearch introduced a breaking change in 7.14
-                            # which makes it incompatible with previous major versions.
+                            "==7.15.0",
+                            "<8",
+                            "==8.0.1",  # 8.0.0 has a bug that interferes with tests
+                            latest,
                         ]
                     },
                 ),
-                Venv(pys=select_pys(), pkgs={"elasticsearch1": ["~=1.10.0"]}),
-                Venv(pys=select_pys(), pkgs={"elasticsearch2": ["~=2.5.0"]}),
-                Venv(pys=select_pys(), pkgs={"elasticsearch5": ["~=5.5.0"]}),
-                Venv(pys=select_pys(), pkgs={"elasticsearch6": ["~=6.8.0", latest]}),
-                Venv(pys=select_pys(), pkgs={"elasticsearch7": ["~=7.11.0"]}),
+                Venv(pys=select_pys(), pkgs={"elasticsearch1": latest}),
```

Comment:
These versions should probably stay pinned to the earliest supported version in accordance with the [compatibility policy](https://github.com/DataDog/dd-trace-py/blob/46466889078eea0b7cf454b06f7456ee6be37c7a/docs/contributing-integrations.rst#what-versions-of-a-given-library-should-an-integration-support).

Reply:
That's exactly what I based the changes here on, did I misread it? 😅

```
For each third-party library ddtrace provides integrations for:
- Test the latest major version. Specifically, test the min-max of the latest major version (the oldest and newest minor releases from the major version) going back 2 years.
- Test the latest minor release of non-latest major versions going back 2 years.
- If no new releases within the past 2 years, test the latest version.
- If support for legacy Python versions (2.7, 3.5, 3.6) was dropped in a major release, then test the latest minor release to support the legacy Python version only.
```

which I interpreted as:

- min-max of 8 (`8.0.0` has a bug that fails on a sort like `name:desc`, so I skipped that with a note)
- for 7, `7.15` was released in Sep 2021
- for others, test the latest version

I wasn't sure whether I should consider those versioned modules as separate integrations when applying these criteria? i.e. test min-max of all of those? Sounds like that's what you're saying, if so I should test min _and_ max of each of those, not just min.

Elasticsearch also has the fun quirk that `7.14+` will refuse to connect to unofficial server versions like AWS OpenSearch, so likely a lot of folks are pinned to `7.13.4`. If we're adding more versions to test we should probably include that one.

Reply:
For most of these it turns out `latest` was already a synonym for the previous requirement, but I restored to what they were. Added `7.13` for the pinned folks out there